main.mjs

```diff
@@ -28,6 +28,7 @@ function renderChart(chartData) {
       vAxis: { minValue: 0 },
       hAxis: { format: 'MMM d' },
       isStacked: true,
+      colors:['#3366cc', '#dc3912', '#ff9900', '#aaaaaa'],
```

Comment:
Had to explicitly set the colors so that the out of scope tests don't get a green color. I re-used the same RGB values as already set for the existent bars.